files/storage.py

```diff
@@ -7,6 +7,21 @@
 
 
 class FileHashedNameMixin:
+    # Workaround for modules that do not completely support Django 4.
+    # TODO: Remove this while adding Django 4.0 support.
```

Comment:
I don't understand this TODO comment very well, we should remove this while adding Django 4.0 support? What does that mean? Can you clarify please?